templates/outer-loop/Dockerfile

```diff
@@ -19,12 +19,13 @@ RUN cd /work/outer-loop-app && \
     mvn -e liberty:create package
 
 # Process any resources or shared libraries - if they are present in the dependencies block for this project (there may be none potentially)
-# test to see if each is present and move to a well known location for later processing in the next stage
-# 
+# test to see if each is present and move to a well known location for later processing in the next stage.
 RUN cd /work/outer-loop-app/target/liberty/wlp/usr/servers && \
     if [ -d ./*/lib ]; then mv ./*/lib /work/configlibdir; fi && \
     if [ ! -d /work/configlibdir/lib ]; then mkdir /work/configlibdir/lib; fi && \
     mv -f */* /work/config/ && \
+    rm -rf /work/config/workarea && \
+    chmod -R 770 /work/config/apps && \
```

Comment:
I'm curious what led you to these two lines.    The .../workarea just has the **.sLock** file, right?    Was that getting in the way somehow?

Maybe regardless it is easier to just `rm` it so we don't copy it into place within the 2nd stage.   We're really only trying to copy config & libraries.  

Typically there'd be nothing in **/work/config/apps** though... since there's no deploy typically done in `mvn liberty:create package`.    So what led you to chmod it ?

Reply:
re: rm -rf /work/config/workarea: 
This is not getting in the way of anything. It is just confusing and potentially an issue to have two OL workareas. It seemed to me as something that should be removed regardless.

sh-4.4$ ls -la /opt/ol/wlp/usr/servers/defaultServer/
...
drwxr-x---. 2 default root  20 Mar 25 22:33 workarea (copied)

sh-4.4$ ls -la /output
...
drwxr-x---. 5 1000580000 root 115 Mar 29 16:37 workarea (created/used)

re: chmod -R 770 /work/config/apps: 
Without this update using (OPENJ9_SCC=false), you get a warning in the logs and an ffdc. The application starts just fine because /opt/ol/wlp/usr/servers/defaultServer/apps/intro.war is there, but it appears that the application expansion attempt must have failed because you get the follwoing warning/trace.

[3/29/21, 16:37:16:673 UTC] 00000035 bm.ws.app.manager.war.internal.WARDeployedAppInfoFactoryImpl E CWWKZ0131W: Could not expand the application intro because of the exception Target [ /opt/ol/wlp/usr/servers/defaultServer/apps/expanded/intro.war ] does 
not exist.

Stack Dump = java.io.IOException: Target [ /opt/ol/wlp/usr/servers/defaultServer/apps/expanded/intro.war ] does not exist
        at com.ibm.ws.app.manager.war.internal.ZipUtils.unzip(ZipUtils.java:280)
        at com.ibm.ws.app.manager.war.internal.ZipUtils.unzip(ZipUtils.java:256)
        at com.ibm.ws.app.manager.war.internal.WARDeployedAppInfoFactoryImpl.expand(WARDeployedAppInfoFactoryImpl.java:79)
        at com.ibm.ws.app.manager.war.internal.WARDeployedAppInfoFactoryImpl.createDeployedAppInfo(WARDeployedAppInfoFactoryImpl.java:107)
        at com.ibm.ws.app.manager.war.internal.WARDeployedAppInfoFactoryImpl.createDeployedAppInfo(WARDeployedAppInfoFactoryImpl.java:35)
        at com.ibm.ws.app.manager.war.internal.WARApplicationHandlerImpl.setUpApplicationMonitoring(WARApplicationHandlerImpl.java:50)
        at com.ibm.ws.app.manager.internal.statemachine.StartAction.execute(StartAction.java:139)
        at com.ibm.ws.app.manager.internal.statemachine.ApplicationStateMachineImpl.enterState(ApplicationStateMachineImpl.java:1317)
        at com.ibm.ws.app.manager.internal.statemachine.ApplicationStateMachineImpl.run(ApplicationStateMachineImpl.java:897)
        at com.ibm.ws.threading.internal.ExecutorServiceImpl$RunnableWrapper.run(ExecutorServiceImpl.java:239)
        at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)

Reply:
Thx for looking into that.